src/kili/utils/labels/geojson/bbox.py

```diff
@@ -0,0 +1,208 @@
+            [vertex_name_to_value["top_right"]["x"], vertex_name_to_value["top_right"]["y"]],
+            [vertex_name_to_value["top_left"]["x"], vertex_name_to_value["top_left"]["y"]],
+            [vertex_name_to_value["bottom_left"]["x"], vertex_name_to_value["bottom_left"]["y"]],
+        ]
+    ]
+
+    return ret
+
+
+def kili_bbox_annotation_to_geojson_polygon_feature(
+    bbox_annotation: Dict[str, Any], job_name: Optional[str] = None
+) -> Dict[str, Any]:
+    """Convert a Kili bounding box annotation to a geojson polygon feature.
+
+    Args:
+        bbox_annotation: a Kili bounding box annotation:
+        job_name: the name of the job to which the annotation belongs.
+
+    Returns:
+        A geojson polygon feature.
+
+    !!! Example
+        ```python
+        >>> bbox = {
+            'children': {},
+            'boundingPoly': [
+                    {
+                        'normalizedVertices': [
+                            {'x': 0.0, 'y': 0.0},
+                            {'x': 0.0, 'y': 1.0},
+                            {'x': 1.0, 'y': 1.0},
+                            {'x': 1.0, 'y': 0.0}
+                        ]
+                    }
+                ],
+            'categories': [{'name': 'A'}],
+            'mid': 'mid_object',
+            'type': 'rectangle'
+        }
+        >>> kili_bbox_annotation_to_geojson_polygon_feature(bbox, 'job_name')
+        {
+            'type': 'Feature',
+            'geometry': {
+                'type': 'Polygon',
+                'coordinates': [
+                    [
+                        [0.0, 0.0],
+                        [0.0, 1.0],
+                        [1.0, 1.0],
+                        [1.0, 0.0],
+                        [0.0, 0.0]
+                    ]
+                ]
+            },
+            'id': 'mid_object',
+            'properties': {
+                'kili': {
+                    'categories': [{'name': 'A'}],
+                    'children': {},
+                    'type': 'rectangle',
+                    'job': 'job_name'
+                }
+            }
+        }
+        ```
+    """
+    bbox = bbox_annotation
+    assert bbox["type"] == "rectangle", f"Annotation type must be `rectangle`, got: {bbox['type']}"
+
+    ret = {
+        "type": "Feature",
+        "geometry": kili_bbox_to_geojson_polygon(bbox["boundingPoly"][0]["normalizedVertices"]),
+    }
+    if "mid" in bbox:
+        ret["id"] = bbox["mid"]
+    ret["properties"] = {
+        "kili": {k: v for k, v in bbox.items() if k not in ["boundingPoly", "mid"]}
+    }
+    if job_name is not None:
+        ret["properties"]["kili"]["job"] = job_name
+    return ret
+
+
+def geojson_polygon_feature_to_kili_bbox_annotation(polygon: Dict[str, Any]) -> Dict[str, Any]:
+    """Convert a geojson polygon feature to a Kili bounding box annotation.
+
+    Args:
+        polygon: a geojson polygon feature.
```

Comment:
not true, this requires custom fields in `properties`. We should choose if we require these fields or not. If we do, this should be clearly stated in the doc. IMHO we should require it, even if this limits the usability of the function.

This comment applies to all the geojson_... methods.

Reply:
I modified the docstrings to explain that the kili property must be set with the categories, etc.